packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -71,6 +79,18 @@ export class GuxAvatar {
   @Prop()
   notifications: boolean = false;
 
+  /**
+   * Show photo change overlay on hover, only available for large avatars
+   */
+  @Prop()
+  changePhoto: boolean = false;
+
+  /**
+   * Occurs when the item has been clicked when changePhoto is true
+   */
+  @Event()
+  edit: EventEmitter<string>;
```

Comment:
Not sure if there's a better name for this. Suggestions welcome

Reply:
editPhoto?